libc/newhdrgen/yaml_to_classes.py

```diff
@@ -22,9 +22,9 @@
 from class_implementation.classes.object import Object
 
 
-def yaml_to_classes(yaml_data):
+def yaml_to_classes(yaml_data, entry_points=None):
```

Comment:
the changes adding support for entrypoint filtering should be a separate patch, and you should also make sure to test it